perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -40,6 +42,8 @@
 	 * Creates a new group in the specific VO defined by object vo in parameter.
```

Comment:
Please update docs for this. We should state, that this combination creates only top-level group in VO.
Also in order to work ``parentGroupId`` param of ``group`` must not be present or must be set to ``null``.